Dockerfile

```diff
@@ -18,8 +18,10 @@ RUN rm ./target/release/deps/vaultwarden_ldap*
 COPY src ./src
 RUN cargo build --release
 
-FROM rust:$RUN_TAG
+FROM ubuntu:focal
 WORKDIR /app
+RUN apt update -y
+RUN apt install -y libssl-dev --no-install-recommends
```

Comment:
Lost a bunch of feedback here, but you should follow recommendations as per `hadolint`. Install `pre-commit` and then run `make check` and `make install-hooks` so that it is run automatically when committing.

Reply:
Just took a quick `yay -S pre-commit hadolint-bin`